node/Cargo.toml

```diff
@@ -12,50 +12,90 @@ edition = "2018"
-sc-keystore = { git = "https://github.com/paritytech/substrate", package = "sc-keystore" }
-sp-inherents = { git = "https://github.com/paritytech/substrate", package = "sp-inherents" }
-sc-transaction-pool = { git = "https://github.com/paritytech/substrate", package = "sc-transaction-pool" }
-sc-transaction-pool-api = { git = "https://github.com/paritytech/substrate", package = "sc-transaction-pool-api" }
-sc-consensus-aura = { git = "https://github.com/paritytech/substrate", package = "sc-consensus-aura" }
-sp-consensus-aura = { git = "https://github.com/paritytech/substrate", package = "sp-consensus-aura" }
-sp-consensus = { git = "https://github.com/paritytech/substrate", package = "sp-consensus" }
-sc-consensus = { git = "https://github.com/paritytech/substrate", package = "sc-consensus" }
-sc-finality-grandpa = { git = "https://github.com/paritytech/substrate", package = "sc-finality-grandpa" }
-sp-finality-grandpa = { git = "https://github.com/paritytech/substrate", package = "sp-finality-grandpa" }
-sc-client-api = { git = "https://github.com/paritytech/substrate", package = "sc-client-api" }
-sp-runtime = { git = "https://github.com/paritytech/substrate", package = "sp-runtime" }
-sp-timestamp = { git = "https://github.com/paritytech/substrate", package = "sp-timestamp" }
-
-# These dependencies are used for the node's RPCs
-jsonrpc-core = "18.0.0"
-sc-rpc = { git = "https://github.com/paritytech/substrate", package = "sc-rpc" }
-sp-api = { git = "https://github.com/paritytech/substrate", package = "sp-api" }
-sc-rpc-api = { git = "https://github.com/paritytech/substrate", package = "sc-rpc-api" }
-sp-blockchain = { git = "https://github.com/paritytech/substrate", package = "sp-blockchain" }
-sp-block-builder = { git = "https://github.com/paritytech/substrate", package = "sp-block-builder" }
-sc-basic-authorship = { git = "https://github.com/paritytech/substrate", package = "sc-basic-authorship" }
-substrate-frame-rpc-system = { git = "https://github.com/paritytech/substrate", package = "substrate-frame-rpc-system" }
-pallet-transaction-payment-rpc = { git = "https://github.com/paritytech/substrate", package = "pallet-transaction-payment-rpc" }
-pallet-contracts-rpc = { git = "https://github.com/paritytech/substrate", package = "pallet-contracts-rpc" }
-
-# These dependencies are used for runtime benchmarking
-frame-benchmarking = { git = "https://github.com/paritytech/substrate", package = "frame-benchmarking" }
-frame-benchmarking-cli = { git = "https://github.com/paritytech/substrate", package = "frame-benchmarking-cli" }
-
-canvas-runtime = { version = "0.1.0", path = "../runtime" }
-
 [build-dependencies]
-substrate-build-script-utils = { git = "https://github.com/paritytech/substrate", package = "substrate-build-script-utils" }
+substrate-build-script-utils = { git = 'https://github.com/paritytech/substrate', branch = "cmichi-tmp-experiment" }
 
 [[bin]]
-name = "canvas"
-path = "src/main.rs"
+name = 'canvas-node'
```

Comment:
This results in the binary name changing from `canvas` to `canvas-node`. The CI currently fails because it still looks for the old-style name:
```
$ cp ${CARGO_TARGET_DIR}/release/canvas ./artifacts/canvas-linux/canvas
cp: cannot stat '/ci-cache/canvas-node/targets/73/build-linux/release/canvas': No such file or directory
```
Is there any reason for changing it to `canvas-node`? I would otherwise opt for keeping it `canvas`.

Reply:
I second that. The name change is unwarranted. But maybe the repository should be renamed to only `canvas`. The polkadot repository isn't named `polkadot-node`.

Reply:
I prefer `canvas` too

Reply:
> This results in the binary name changing from `canvas` to `canvas-node`. The CI currently fails because it still looks for the old-style name:

🤦